Applications.qmd

```diff
@@ -5,14 +5,45 @@ listing:
   id: Applications
   contents:
     - Applications/*.qmd
-  sort: "date desc"
   filter-ui: true
   page-size: 8
   fields: [title, description]
 ---
 
 
-This section provides an overview of the EO Applications available from Copernicus Data Space Ecosystem.
+Copernicus Data Space Ecosystem provides a wide range of applications that can be used to access, process, and visualize Copernicus data. These applications are designed for users of different skill level to be used for various applications.
+
+For new Copernicus dataspace ecosystem users we advise to explore the [Copernicus Data Space Browser](Applications/Browser.qmd) as starting point. In this platform users can explore various datasets, employing tools for visualization, comparison, and downloading with ease. 
+
+![](Applications/_images/SearchResult.png)
+
+
+Advanced users interested in designing custom EO processing workflows can explore the [JupyterLab environment](Applications/JupyterHub.qmd) for a more interactive analysis. Within the Jupyterlab environment, users can execute Python code and access data interactively.
+
+![](Applications/_images/JupyterLab_welcome.png)
+
+Moreover, these jupyterlab environments support the use of openEO to acces and proces the data in an interactive manner. More information on the openEO API can be found [here](APIs/openEO/openEO.qmd). Nevertheless, for users who are intersted in using openEO API in a GUI environment, we recommend exploring the [openEO Web Editor](Applications/WebEditor.qmd) documentation. This tool allows users to interact with the openEO API in a more visual manner.
+
+
+![](Applications/_images/webeditor/webeditor_result.PNG)
+
+The Copernicus Dataspace Ecosystem supports FAIR and Open Science principles through sharing EO algorithms. The [openEO Algorithm Plaza](Applications/AlgorithmPlaza.qmd) documentation introduces the plaza. It explains how one can use the platform to share and use EO algorithms.
+
+
+![](Applications/_images/PlazaOverview.png)
+
+Furthermore, the ecosystem encompasses a [QGIS Plugin](Applications/QGIS.qmd) designed to facilitate access and processing of EO data. The documentation for the QGIS Plugin offers an overview of its functionality and usage within the QGIS platform.
```

Comment:
isn't qgis mainly used for visualisation?

Reply:
indeed

Reply:
so why do we currently describe it as a processing tool?

Reply:
Qgis is a processing tool as well, along with Visualisation. However, the QGIS Plugin provided by Sentinel Hub is indeed for visualisation.